packages/libs/components/src/map/MapVEuMap.tsx

```diff
@@ -154,6 +154,10 @@ export interface MapVEuMapProps {
   /* selectedMarkers setState (for on-click reset) **/
   setSelectedMarkers?: React.Dispatch<React.SetStateAction<string[]>>;
   children?: React.ReactNode;
+  /* close left-side panel when map events happen */
```

Comment:
We shouldn't be documenting behaviour of the map code here. The callbacks can do *anything*!

Reply:
Thank you for your review and suggestions @bobular I got your point and remove it.